Pollo/Views/Cards/MCResultCell.swift

```diff
@@ -150,75 +137,107 @@ class MCResultCell: UICollectionViewCell {
-        optionLabel.text = resultModel.option
-        percentSelected = resultModel.percentSelected
-        numSelectedLabel.text = "\(Int(percentSelected * 100))%"
         self.correctAnswer = correctAnswer
         self.userRole = userRole
-        let answer = intToMCOption(resultModel.choiceIndex)
-        switch userRole {
-        case .admin:
-            containerView.backgroundColor = correctAnswer == answer ? .lightGreen : .lightGrey
-            containerView.layer.borderWidth = containerViewBorderWidth
-            containerView.layer.cornerRadius = containerViewCornerRadius
-            highlightView.isHidden = true
-            numSelectedLabel.isHidden = false
-            dotView.isHidden = true
-            optionLabel.textColor = .black
-        case .member:
-            containerView.backgroundColor = .clear
-            containerView.layer.borderWidth = highlightViewBorderWidth
-            containerView.layer.cornerRadius = highlightViewCornerRadius
-            optionLabel.textColor = .mediumGrey
-            numSelectedLabel.isHidden = true
-            dotView.isHidden = false
-            let isSelected = resultModel.isSelected
-            selectedDotView.isHidden = !isSelected
-            selectedImageView.isHidden = selectedDotView.isHidden
-            if let correctAnswer = correctAnswer, !correctAnswer.isEmpty {
-                if answer == correctAnswer {
-                    if isSelected {
-                        selectedDotView.backgroundColor = .clear
-                        selectedImageView.image = UIImage(named: correctImageName)
-                    }
-                    showCorrectAnswer = true
-                    highlightView.backgroundColor = .lightGreen
-                    highlightView.layer.borderColor = UIColor.polloGreen.cgColor
-                } else {
-                    if isSelected {
-                        selectedDotView.backgroundColor = .clear
-                        selectedImageView.image = UIImage(named: incorrectImageName)
-                    }
-                    highlightView.backgroundColor = .lightGrey
-                    highlightView.layer.borderColor = UIColor.coolGrey.cgColor
-                }
-            } else {
-                if isSelected {
-                    selectedDotView.backgroundColor = .coolGrey
-                    selectedImageView.image = nil
-                }
-                highlightView.backgroundColor = .lightGrey
-                highlightView.layer.borderColor = UIColor.coolGrey.cgColor
+        
+        numSelectedLabel.isHidden = userRole == .member
+        numSelectedLabel.text = "\(resultModel.numSelected)"
+        
+        optionLabel.text = resultModel.option
+        optionLabel.textColor = userRole == .admin ? .black : .mediumGrey
+        
+        percentSelected = resultModel.percentSelected
+        percentSelectedLabel.isHidden = userRole == .member
```

Comment:
Can you sync up with design and check if we want to include the percentages/counts for the student-side?